nixos/modules/programs/fish.nix

```diff
@@ -29,6 +29,15 @@ in
         type = types.bool;
       };
 
+      package = mkOption {
```

Comment:
Is there a specific need to introduce this option? See also #50476

Reply:
Thanks for the pointer, removed.